server/api/submissions.js

```diff
@@ -66,7 +61,7 @@ router.get('/:teamID/:submissionID', async (req, res) => {
  *     tags:
  *       - submissions
  */
-router.post('/:teamID/create', (req, res) => {
+router.post('/create', (req, res) => {
```

Comment:
I think we can make this endpoint just `/`, so when someone creates a submission they hit `POST /submissions` and that should be good enough